build/test.ps1

```diff
@@ -52,7 +62,7 @@ function Test-Python {
     }
 }
 
-Test-One '../iqsharp.sln'
+Test-One '../iqsharp.sln' @("AzureClient", "IQSharpEngine", "Workspace")
```

Comment:
This feels a bit fragile. I think we're going to forget to add the new classes to the script later.
Maybe a little more work, but what if we just break the tests into multiple projects?

Reply:
All tests are run regardless of the list of filters here. We run `dotnet test` with each of these filters, followed by `dotnet test` for tests that didn't match any of the filters:

https://github.com/microsoft/iqsharp/blob/42b29c2054e7a552acf238a5ced71235075b3add/build/test.ps1#L15-L18

I'll add comments to the code to make it more clear what is happening.

Yes, this still has the problem that if we add _enough_ new tests, we may run into the out-of-memory issue again unless we add more filters here. But we don't run the risk of missing tests.

I've opened #224 to track fixing the underlying issue here, so that we don't forget about it.